docs/_docs/installation/macos.md

```diff
@@ -3,155 +3,28 @@ title: Jekyll on macOS
 permalink: /docs/installation/macos/
 ---
 
-## Install Command Line Tools
```

Comment:
Homebrew automatically installs the command line tools if they're not already there, and since we're installing Ruby via `chruby`, which in turn gets installed via Homebrew, this step is redundant.